cupy/_core/fusion.pyx

```diff
@@ -119,7 +119,15 @@ class _FusionVarCUDA(object):
         elif isinstance(val, complex):
             init = '({}, {})'.format(c.real, c.imag)
         elif isinstance(val, (int, float)):
-            init = '= {}'.format(c)
+            if numpy.isnan(val):
```

Comment:
(As discussed offline) there are constants defined in CUDA header file, but is not included by default so I think it's ok to remain as-is.
https://github.com/cupy/cupy/blob/master/cupy/_core/include/cupy/math_constants.h

Btw, the variable could be a double?